src/ORM/phpstan.neon.dist

```diff
@@ -24,3 +24,4 @@ parameters:
 		- '#^Parameter \#1 \$fields of method Cake\\Datasource\\EntityInterface::.*\(\) expects list<string>, array<string> given\.$#'
 		- '#^Parameter \#1 \$field of method Cake\\Datasource\\EntityInterface::has\(\) expects list<string>\|string, .+<string> given\.$#'
 		- '#^Parameter \#1 \$field of method Cake\\Datasource\\EntityInterface::unset\(\) expects list<string>\|string, array<string>\|string given\.$#'
+		- '#^Call to an undefined method Cake\\Datasource\\EntityInterface::setPatchable\(\)\.$#'
```

Comment:
I have no idea why PHPStan now doesn't understand that method exists on the EntityInterface even though it had no problem previously

Reply:
Rebasing solved this. Seems this was an issue with an earlier phpstan version